src/pages/GetStarted/GetStartedCauses.vue

```diff
@@ -1,31 +1,191 @@
 <template>
-	<div>
-		<kv-progress-bar value="33" max="100" />
-		<div class="row page-content">
-			<div class="small-12 large-5 columns">
-				<h1>Choose causes <br class="lu">you care about</h1>
-				<p>
-					With Kiva, you can choose causes you care about and borrowers to support.
-					You’ll get updates from borrowers as they repay their loan.
-				</p>
+	<div class="causes">
+		<kv-progress-bar class="progress-bar" value="33" max="100" />
+		<form
+			@submit.prevent="onSubmitForm"
+			:disabled="selectedCauses.length === 0"
+		>
+			<div class="page-content">
+				<fieldset>
+					<div class="row">
+						<div class="small-12 xlarge-5 columns get-started__intro">
+							<h1>Choose causes <br class="xlu">you care about</h1>
+							<p>
+								With Kiva, you can choose causes you care about and borrowers to support.
+								You’ll get updates from borrowers as they repay their loan.
+							</p>
+						</div>
+						<div class="small-12 xlarge-7 columns">
+							<ul class="get-started__list">
+								<li
+									class="get-started__list-item"
+									v-for="cause in causeList"
+									:key="`cause-${cause.id}`"
+								>
+									<kv-cause-selector
+										class="get-started__cause-selector"
+										:cause="cause.name"
+										:checked="cause.checked"
+										@change="onChangeCauseSelection($event, cause.id)"
+									/>
+								</li>
+							</ul>
+							<p
+								class="get-started__summary text-center"
+								v-html="summaryText"
+							></p>
+						</div>
+						<kv-button
+							class="get-started__submit-btn"
+							type="submit"
+							:disabled="selectedCauses.length === 0"
+						>
+							Next
+						</kv-button>
+					</div>
+				</fieldset>
 			</div>
-			<div class="small-12 large-6 large-offset-1 columns">
-				cause checkboxes go here
-			</div>
-		</div>
+		</form>
 	</div>
 </template>
 
 <script>
+import cookieStore from '@/util/cookieStore';
+// import gql from 'graphql-tag';
+
+import KvButton from '@/components/Kv/KvButton';
+import KvCauseSelector from '@/components/Kv/KvCauseSelector';
 import KvProgressBar from '@/components/Kv/KvProgressBar';
 
 export default {
 	components: {
+		KvButton,
+		KvCauseSelector,
 		KvProgressBar,
 	},
 	metaInfo: {
 		title: 'Get Started'
 	},
+	inject: ['apollo'],
+	data() {
+		return {
+			causeList: [
+				{
+					id: 5,
+					name: 'women',
+					checked: false,
+				},
+				{
+					id: 6,
+					name: 'shelter',
+					checked: false,
+				},
+				{
+					id: 4,
+					name: 'education',
+					checked: false,
+				},
+				{
+					id: 0, // TODO: Couldn't find an id for technology
+					name: 'technology',
```

Comment:
@emuvente Do you know what the id is for "technology" category?

Reply:
I can't find one for htat either, so I think we'll have to make it.

Reply:
I think we need to figure out what the loan search criteria is supposed to be for this. Then if you do create a new Loan Channel, please add it to the query map too (https://github.com/kiva/ui/blob/master/src/plugins/loan-channel-query-map.js)

Reply:
I believe for the technology category it's just the "Technology" tag